driver/examples/configs/README.md

```diff
@@ -0,0 +1,45 @@
+# Driver Configurations
```

Comment:
This is good to have for now, in the long term it will be hard to maintain this README separate from the code. If and when we put together documentation, we should try to make it so Sphinx automatically documents these details based on dataclass docstrings.